packages/expo-video/ios/VideoManager.swift

```diff
@@ -10,15 +10,38 @@ import ExpoModulesCore
 class VideoManager {
   static var shared = VideoManager()
 
+  private var managerQueue = DispatchQueue(label: "com.expo.video.manager.managerQueue")
   private var videoViews = NSHashTable<VideoView>.weakObjects()
   private var videoPlayers = NSHashTable<VideoPlayer>.weakObjects()
 
-  func register(videoPlayer: VideoPlayer) {
-    videoPlayers.add(videoPlayer)
+  func register(videoPlayer: VideoPlayer, queue: DispatchQueue? = nil) {
```

Comment:
Do we really need a way to use a custom queue?

Reply:
For now no, I just wanted to future-proof it 😅